### PR TITLE
Replace MySQL with Neo4j

### PR DESCRIPTION
**Pull request by giang-nghg**

### CODE REVIEW

**Comment by giang-nghg** [comment]
Fixes #1

File: swagger_server/controllers/default_controller.py [hunk 1]
```diff
@@ -19,12 +18,13 @@ def distance_post(body):  # noqa: E501
...
-        ).all()
+        neighbors = SampleNode.nodes.filter(name=sample.experiment_id)\
+            .neighbors.match(dist__lt=10)
 
         return ApiResponse(
             type='distance',
             sub_type='nearest-neighbor',
-            result={sample.s2: sample.d for sample in within_distance}
+            result=[{
+                'name': s.name
```
Comment (Zhicheng-Liu) [inline swagger_server/controllers/default_controller.py:28]: Can we also return the distance as well. See the format specification in https://trello.com/c/wDicpcGt

File: swagger_server/__main__.py [hunk 2]
```diff
@@ -1,53 +1,31 @@
...
+def create_dummy_data():
+    samples = []
+    for name in ['s1', 's2', 's3']:
+        try:
+            s = SampleNode.nodes.get(name=name)
+        except SampleNode.DoesNotExist:
+            s = SampleNode(name=name).save()
+        samples.append(s)
+    samples[0].neighbors.connect(samples[1], {'dist': 5})
+    samples[0].neighbors.connect(samples[2], {'dist': 10})
```
Comment (Zhicheng-Liu) [inline swagger_server/__main__.py:21]: Does this mean `s1` have 2 neighbours `s2` and `s3`? But when I searched for 's1', I got only 's2' in the result.

**Review by Zhicheng-Liu — CHANGES_REQUESTED** [review]

File: swagger_server/__main__.py (hunk 2 above)
Comment (giang-nghg) [reply swagger_server/__main__.py:21]: You mean using Python (`SampleNode.get(name='s1').neighbors`)?

File: swagger_server/controllers/default_controller.py (hunk 1 above)
Comment (giang-nghg) [reply swagger_server/controllers/default_controller.py:28]: sure

File: swagger_server/__main__.py (hunk 2 above)
Comment (Zhicheng-Liu) [reply swagger_server/__main__.py:21]: No, I mean when I calling the API served by the docker container (with this dummy data).

Comment (Zhicheng-Liu) [reply swagger_server/__main__.py:21]: `C02W11SKHV2R:paste_vcf zl$ curl --url http://192.168.99.101:8080/distance   --header 'content-type: application/json'   --data '{
"experiment_id": "s1"
}'
{
  "result": [
    {
      "name": "s2"
    }
  ],
  "subType": "nearest-neighbor",
  "type": "distance"
}
`

Comment (giang-nghg) [reply swagger_server/__main__.py:21]: Ah, the prototype API returns the neighbours that's has distance < 10: https://github.com/iqbal-lab-org/dist-api-prototype/pull/2/files#diff-62857f1e2c6c9a1955e06fd24eddd186R22

**Review by Zhicheng-Liu — APPROVED** [review]
Looks OK for prototype.

**Pull request merged**